Week2/Bank.java

```diff
@@ -0,0 +1,118 @@
+package Week2;
+
+import java.util.*;
+import java.math.BigDecimal;
+//  계좌생성, 출금, 입금, 송금 메서드 구현
+// 내부에서 입력을 받는 액션?
+// 적금 계좌는
+public class Bank {
+    private ArrayList<Account> accountArray;

```

Comment:
자료구조를 직접 변수명으로 만드는 것은 좋지 않습니다. 아래와 같이 변경하는 것은 어떨까요?
```suggestion
    private ArrayList<Account> accounts;

```